server/src/queues/deployApp.ts

```diff
@@ -0,0 +1,104 @@
+import { Worker, Queue } from 'bullmq';
+import createDebug from 'debug';
+import Redis from 'ioredis';
+import { pubsub } from './../index';
+import { config } from '../config';
+import { sshConnect } from '../lib/ssh';
+import { dokku } from '../lib/dokku';
+
+const queueName = 'deploy-app';
+const debug = createDebug(`queue:${queueName}`);
+const redisClient = new Redis(config.redisUrl);
+
+interface QueueArgs {
+  appName: string;
+  gitRepoUrl: string;
+  branchName?: string;
+  dokkuApp?: boolean;
+  appId?: string;
```

Comment:
let's just take the `appId` as argument as we can fetch the rest of the data from the db